### DISCUSSION TITLE
Contributor License Agreement

### DISCUSSION THREAD

**Discussion by jjbursik** [jupyter/enterprise_gateway#1080] [Q&A] [answered]
Target is considering making a contribution to the [Jupyter Enterprise Gateway repo](https://github.com/jupyter-server/enterprise_gateway).  Does Jupyter require contributors to sign a contributor license agreement?

**Comment by lresende** [comment, marked as answer]
No, there is no need to sign CLA or any other agreement. Please feel free to submit contributions via PRs.
